data/campaigns/Under_the_Burning_Suns/scenarios/08_Out_of_the_Frying_Pan.cfg

```diff
@@ -205,6 +205,12 @@
     # add more dirt at easier difficulties
     # set starting scenario objectives
 
+#define PASSABLE_HEX
+    # Adds passable attribute to a unit
+    [+unit]
+        passable=yes
```

Comment:
Better to put it as:
```ini
placement=map
passable=yes
```

> use placement=map and passable=yes instead.

<https://wiki.wesnoth.org/SingleUnitWML#Creation_Options>

Reply:
Isn't that redundant? You already need x and y for the {UNIT} macro so placement would just put them on the map somewhere passable, which is what passable=yes does. Please expatiate.

Reply:
> Isn't that redundant? You already need x and y for the {UNIT} macro so placement would just put them on the map somewhere passable, which is what passable=yes does. Please expatiate.

No, that's the currently used one. the deprecated ones are `map_passable` and `leader_passable`.
The wiki mentioned both have to there (Notice the "And") so pointed it out.

Reply:
But I am not using `map_passable` nor `leader_passable` inside the `placement` option, im using the `passable` option. And anyways the wiki says:

> placement: How the unit should be placed: can be one value or a comma-separated list of values. Default value is 'map,leader' for a leader given directly in [side], "" otherwise. By default, 'map,recall' is implicitly appended to the end of the list.

So placement=map is default and does not have to be specified, I think.

Reply:
If that's the case, then I guess it shouldn't be there. I add it in for works though (to avoid any bugs popping up later)

Reply:
> I add it in for my works though (to avoid any bugs popping up later).

Ah, I see, well I think this is a safe bet, anyways if a bug related to this is found later and im still over here, i'll try to fix it.

Reply:
Cool